### ISSUE TITLE
Growth Team Lead

### ISSUE THREAD

**Issue by cbeams** [bisq-network/roles#100]
Docs: https://bisq.wiki/Growth_Team
Teams: @bisq-network/growth, @bisq-network/team-leads

**Comment by m52go** [comment]
## Cycle 12 Report

March was a great month for trading activity on the network—**the best ever**, by a good margin. There were 3,597 trades across the network, and the last record was 3,037 set back in August 2019. It seems the [month's growth initiatives](https://github.com/bisq-network/growth/issues/185) helped moved the needle, but BTC volatility and momentum from February certainly helped.

On the not-so-bright side:
* I overestimated how much work could actually be done in the month of March (e.g., videos, bots), so some of it spilled into April
* month-long initiative might have been too long (seemed to drag on after the second week, but this might have been because of the virus taking over the collective psyche)
  * COVID-19 became a major buzzkill, particularly going in to weeks 3 and 4...it just seemed tone-deaf to push trading in the second half of the month
* Bitcoin 2020 was postponed
* most trading activity was for fiat, so overall volume and revenue was low

In any case, trading activity was still fantastic, and work done during the month will be put to good use...much of it is being weaved into [a new getting-started page](https://github.com/bisq-network/bisq-website/issues/356). The lasting impact of this page should be big. Over 5,000 people visit the current getting-started guide, so even a modest improvement in converting these folks could go a long way.

### Near-term Goals

Once the new getting-started page is in place, I would like to purse the following:

* `volume`: publish altcoin-specific materials for l-btc and xmr
* `volume`: research how to increase fiat trade sizes in major markets
* `volume`: explore jpy potential...quick transfers and high limits on Bisq make this market very appealing (and we have market makers!)
* `publicity`: start publishing informative and interesting articles (monthly cadence to start)
* `publicity`: explore cross-promotion partnerships (?)

I'm also in talks with luisantoniocrag_ about spearheading a more comprehensive push into Latin American markets.

Unfortunately the recent security incident cast a bit of a cloud over the stellar results of March, but it seems trading has largely bounced back already, and we might be able to proceed normally after publishing a thorough post-mortem. Fingers crossed!

### Operations

I'll be switching from my local Wekan instance to GitHub for managing the growth function. The use case I had envisioned didn't pan out (multiple people collaborating simultaneously), so GitHub makes more sense after all. Maybe this mechanism can be revisited as the team grows.

/cc https://github.com/bisq-network/compensation/issues/540

**Comment by m52go** [comment]
## Cycle 13 Report

April activity was still strong in spite of the security incident: there were 2854 trades in April; only 2 other months in Bisq history that have been more active. Volume was almost equal: 471 BTC traded in April v 477 BTC traded in March.

luisantoniocrag_ got started on the Latin American growth effort with MwithM, contributing blog posts, videos, and more.

A new getting-started page is reviewed and ready to be merged. The website was cleaned up and made ready for a better translations workflow.

Bisq made a splash for the halving, which was entirely enabled by Pedro's outstanding work.

Informal discussions continued about improving the new trade protocol, which may be a factor in decreased XMR volume. A post/document is in progress to put thoughts and analysis in one place for better discussion, evaluation, and brainstorming for a better solution.

/cc https://github.com/bisq-network/compensation/issues/572

**Comment by m52go** [comment]
## Cycle 14 Report

May continued April's strength. This last cycle saw the delivery of a new getting-started guide and the rollout of the Bisq wiki, two tools which are instrumental to newcomers.

Now new priorities to build on these resources will be set for Cycle 15 and beyond.
* pursue regular publishing schedule (written/videos) to give people new reasons to use bisq
* experiment with front page redesign to increase download conversion rate
* explore feasibility of adding strings (links to wiki) to increase new user comfort, encourage more trading, and reduce support cases
* iterate reporting project to next stage: provide context to issuance numbers.

These priorities will soon be organized on a new GitHub board.

/cc https://github.com/bisq-network/compensation/issues/604

**Comment by m52go** [comment]
## Cycle 15 Report 

June numbers were quite strong, but July seems to be lagging a bit so far. USPS Money Order volume was at an all-time high in June and will do even better in July, thanks to the guidance (tips and tricks published in [wiki article](https://bisq.wiki/US_Postal_Money_Order)) and market making (handful of attractive offers kept online) of a couple of users.

Priorities have been organized on a [GitHub board](https://github.com/bisq-network/growth/projects/1), along with a new [publishing board](https://github.com/bisq-network/growth/projects/3) for blog posts, videos, etc.

They're largely the same as those outlined last month, but I wasn't able to make substantive progress this cycle on implementing.

/cc https://github.com/bisq-network/compensation/issues/631

**Comment by m52go** [comment]
## Cycle 16 Report 

This cycle I largely focused on growth infrastructure items:
* [translations workflow](https://github.com/bisq-network/bisq-website/pull/386) for website
* reviewing and merging [reporting tool](https://github.com/bisq-network/growth/tree/master/reporting/compensation-bot) for budgeting
* [wireframing a dashboard](https://github.com/bisq-network/projects/issues/41#issuecomment-670926866) for more polished, in-depth public reporting
* establishing a #growth-alerts channel on keybase so the community can keep a closer pulse on trading activity
* started improving strings across the software (both on my own and [with others](https://github.com/bisq-network/bisq/issues/4416)) to refine the user experience; some [pull](https://github.com/m52go/bisq/tree/refresh-strings-1) [requests](https://github.com/m52go/bisq/tree/refresh-strings-2) stand ready, while the rest remain to be done)

Most items above are either complete or in review, meaning I should be able to get back to actual growth efforts that move the needle on trading activity in the upcoming cycle.

I'm also actively trying to recruit help...had calls with a small handful of folks over the past few weeks. Hoping to find folks to at least review website pull requests and produce post and videos.

/cc https://github.com/bisq-network/compensation/issues/649

**Comment by m52go** [comment]
## Cycle 17 Report 

Growth infrastructure items mentioned in the last cycle report are largely complete (or not something I'm actively working on):
* translations workflow is live
* budgeting tool is live
* dashboard is in development
* #growth-alerts channel is live
* strings project is in progress, in collaboration with Bayernatoor, but I'm regarding it as more of a part-time second-tier priority

Now I'd like to launch a series of growth initiatives in the upcoming cycle to actually grow volume and trading activity. 

But instead of putting the whole thing on myself, I'd like to recruit leads for each market we decide to focus on, have them determine tasks and initiatives that are best for their markets, and also have them spearhead implementing the initiatives as well.

To that end, I'll be posting some docs in the growth channel soon as a starting point to kick this off.

/cc https://github.com/bisq-network/compensation/issues/676

**Comment by m52go** [comment]
## Cycle 18 Report 

Doc mentioned in last report [has been created](https://cryptpad.fr/pad/#/2/pad/view/3Zp0c6z0P4gCUw3n0IzNYjXJT3C+Egbajtz2U7rGzC0/). Intention is to use it as a launching point for encouraging more collective growth efforts.

Original goal wasn't to create such a comprehensive doc, but upon interacting with some current and potential contributors, it became obvious that such a doc would be worthwhile.

Plan is to use this doc to create a handful of bounties/issues for discrete tasks (e.g., create press kit, analyze SEO opportunities, etc) and loop in potential contributors in key markets to determine strategies for those markets.

/cc https://github.com/bisq-network/compensation/issues/702

**Comment by m52go** [comment]
## Cycle 19 Report 

Wasn't terribly active in the past cycle, but I'm planning [a growth drive](https://github.com/bisq-network/growth/issues/211) for the first week of January (currently targeting CAD, but might also target new payment methods like TransferWise).

The idea behind the structure of this initiative is to target region-specific publicity channels and create region-specific articles and videos ahead of time so that maximum noise can be made during the week of the initiative.

Creating a [growth kit](https://github.com/bisq-network/growth/issues/204) and [press kit](https://github.com/bisq-network/growth/issues/203) are medium-level priorities currently in progress.

@pazza83 has started off with a bang as payment method maintainer, which may help with growth potential.

**Comment by m52go** [comment]
## Cycle 20 Report 

[CAD and TransferWise growth drive](https://github.com/bisq-network/growth/issues/211) will be taking place in the first week of January. Community-spawned "Bisq Day" is happening on 4 January right after Proof of Keys day on 3 January. It's rather informal and impromptu, so to speak, but we'll see what happens.

Plan is to learn from these efforts and immediately start planning the next one.

Deck for growth kit was done as well.

**Comment by m52go** [comment]
## Cycle 21 Report 

[CAD and TransferWise growth drive](https://github.com/bisq-network/growth/issues/211) flopped...should probably be more active on Keybase to rally community. There seemed to be a modest uptick in community activity in the beginning of January for "Bisq Day" and #proofofkeys, but it seems to have fallen since. 

Trading activity remains strong. Number of trades in January blew past 4,000 (highest ever by a fair margin) and daily average volume is highest it's been in a while.

Lots of new payment methods on deck, courtesy of pazza83.

/cc https://github.com/bisq-network/compensation/issues/775

**Comment by m52go** [comment]
## Cycle 22 Report 

Tried a new call series [Bisq Demo Sessions](https://github.com/bisq-network/growth/issues/240) that got a promising start. This first event got 28 concurrent attendees on the livestream, and over 400 views since, both numbers that far exceed any of the previous growth call formats.

[Trade limits](https://github.com/bisq-network/proposals/issues/295) are a serious issue and need to be addressed soon. This was discussed on the last teamleads call but needs more proactive effort to move the needle. 

Discussion thread on reporting got me thinking of alternative ways to implement reporting, and with more contributors seeming to pop up and show interest in growth-related items, maybe we can muster the manpower to get it done too. Will make a project proposal in Cycle 23.

**Comment by m52go** [comment]
## Cycle 23 Report

Trade limits for account types requiring signing were ultimately kept at 0.01 BTC, but [a warning was put in place](https://github.com/bisq-network/bisq/pull/5344) to warn traders of the risks.

I made a proposal to add [semi-automated DAO reporting to the website](https://github.com/bisq-network/growth/issues/245). I plan to see it implemented in 2 parts, and already got started on the first part.

huey735 started holding Keybase meetings to discuss strategic topics for the project. The [first one](https://github.com/bisq-network/bisq/discussions/5322) covered concepts for moving Bisq off of layer one, the 1-tx trade protocol concept, and (briefly) the new UI. The [next one](https://github.com/bisq-network/bisq/discussions/5378) covered the new UI in more detail, the API, and the 1.6.0 rollout of SegWit for BSQ.

These meetings have gotten high engagement so far from core contributors, and the signal-noise ratio seems high. Hope to see them continue taking place with the focus and participation and effectiveness that they have been.

**Comment by m52go** [comment]
## Cycle 24 Report

A perfect storm of high fees, text-only meetings on Keybase, and new ideas/proposals has snowballed into real momentum on figuring out a way forward for Bisq v2. L-BTC as base currency has gotten the most air-time outside of Bisq, but a modular messageboard-style network ("Misq") seems to have the most momentum within the project right now.

wiz went on Citadel Dispatch to discuss the possible integration of L-BTC as a base currency in Bisq, and might go on again soon to discuss Bisq more (the recent discussion was more about Liquid in general; Bisq wasn't discussed a whole lot).

Payment method research continues. pazza83's plan is to frontload efforts with thorough research and implement those which are considered appropriate in batches before the end of the year.

Bisq's presence at Bitcoin 2021 is shaping up to be quite epic. It's now partnering with mempool.space, so the booth will be shared by both projects. Booth details were just finalized (organizers gave Bisq first choice on booth placement, so the Bisq booth is _right_ at the entrance to the venue), logistics are being arranged, and t-shirts and other swag have been ordered to sell and/or give away at the event.

The semi-automated reporting effort was paused as preparations for Bitcoin 2021 took much of my time.

**Comment by m52go** [comment]
## Cycle 25 Report

Spent much of the cycle on Bitcoin 2021. It was epic.

Traffic at the booth was insane...it was right in front of 8 doors opening the venue to 13,000+ people. A number of people said the Bisq/Mempool booth was the only one worth visiting, and that Bisq's involvement was a highlight of the whole event. Apparently Bisq was mentioned on stage several times, which (in addition to the A+ location) drove people back to the booth several times.

Sold over 500 Bisq and Mempool t-shirts (slightly more Bisq than Mempool; all proceeds went to Mempool since they paid for the shirts).

![IMG_9179](https://user-images.githubusercontent.com/735155/128562771-91ab2a7b-fb81-4b73-a47c-e8fbe7610810.JPG)

![IMG_9177](https://user-images.githubusercontent.com/735155/128562887-7e3326c9-d8c6-4333-91ab-f2105de6d9b6.JPG)

![39D4E2C1-7051-463C-BC5F-28F556E69E94](https://user-images.githubusercontent.com/735155/128562977-abf6e9c3-03b8-4ded-a5bd-6b5ef9c3fd33.jpg)

**Comment by m52go** [comment]
## Cycle 26 Report

Merged a number of long-standing PRs into the website, and [planned a better DAO status page](https://github.com/bisq-network/growth/issues/245#issuecomment-892441070).

**Comment by m52go** [comment]
## Cycle 30 Report

A [better DAO status page](https://bisq.network/dashboard/) was merged earlier this cycle. Page generation is automated, and all details are documented [here](https://github.com/bisq-network/bisq-website/tree/master/.dao).

Bitcoin Magazine began publishing their quarterly paper magazine again this month, and the first issue featured an in-depth look at El Salvador. The back cover features Bisq and Mempool. This back cover is the last remaining item of our conference deal with them.

![btcmag-1](https://user-images.githubusercontent.com/735155/142701256-b22d46c6-fe14-401e-82f7-1165c1d094d6.jpeg)
![btcmag-2](https://user-images.githubusercontent.com/735155/142701264-3e658792-1a58-4fa2-bff5-65070db0dfc1.jpeg)

[[source of images](https://twitter.com/nikcantmine/status/1458829757706084354)]

Otherwise I've become frustrated with the direction of the project. Bisq has largely saturated existing avenues of growth in its current form. It's a wonderful tool for power-users in USA and Europe. Beyond that, it's unlikely to gain significantly more traction, as it's cut off from millions in USA and Europe and billions in emerging markets.

The solution for this is to improve accessibility. The way to do this is to create additional interfaces (e.g. mobile and node-in-a-box availability like Umbrel) and enable integrations with other services (e.g. with other wallets, exchanges, etc).

These things can only happen with an API and a headless version of the core software that can run on a server and/or Raspberry Pi. While efforts to produce an API and daemon have been ongoing for years, and for the past several months in particular, it doesn't seem like the current API is ready for developers to build on. The [mobile app in progress right now](https://github.com/bisq-network/proposals/issues/342) promises to bring such functionality eventually, but it is unclear if the API and daemon will be ready for the mobile app to use. This means that there is a chance that the mobile app may never be capable of being anything but a remote-control for the existing desktop mobile app, which means it may not be capable of growing the user base.

I have met several projects and companies in my travels over the past several months who would love to integrate Bisq into their offerings and effectively extend Bisq's reach overnight, but the lack of software tools make these growth avenues impossible. Payment processors, exchanges, wallets, custody services, merchants...I've had preliminary conversations with many such people under the impression an API and/or Bisq 2.0 would be ready before 2021 ended, but the end of 2021 is almost here and all those people will be left hanging now.

What's left? Education and marketing can only do so much. New payment methods are mostly irrelevant if the target users cannot actually use Bisq in sufficient numbers (e.g. China and India). Altcoin trading isn't something most Bisq users care about. The core function of trading remains vulnerable to fee spikes, and a new protocol doesn't appear to be on the horizon.

In a nutshell, I don't see the project growing meaningfully (i.e. order-of-magnitude steps) without major changes. So I don't want to be held responsible for growth anymore. I have trimmed my other roles as well and generally expect to be less available across the project going forward.

@cbeams please unassign me from this role.

**Comment by pazza83** [comment]
Hi @m52go sorry to hear you have become frustrated.

I agree that Bisq is a great tool for power users. I don't think that this needs to be limited to USD/EUR. I think Bisq just lacks market makers in other markets. Eg there is no reason I see that power users in places like Canada and Australia could not develop markets to sustainable levels. I think Bisq does not communicate a use case for such users. It is left for users to discover the possibilities themselves. 

I agree that improving accessibility to users for small trades and those in emerging markets, without access to a computer, should be a major goal. 

The development of API, mobile, Bisq 2.0 does take a lot of time, and progress as a whole can seem slow. I attribute this partially to the decentralized nature of Bisq, which I also see as one of it's major strengths, important for the long term sustainability of the project.

I think you have done a fantastic job with the development of education and marketing of the project. From a personal perspective I really enjoyed learning about Bisq through listening to you on youtube and various podcasts. I was left with a great impression of Bisq from the way you spoke about the direction of the project, the creation of the DAO and the goals Bisq was trying to achieve. The documentation you have done is also excellent. I am sure many people, myself being one, were encouraged to give Bisq a try from listening to you. Many thanks for this.

I would love to see the project growing in orders of magnitude. I feel optimistic this will happen, but agree it will need the major changes you mention to achieve this. I think it would be unfair to hold anyone person accountable for these changes. It will take a great team to achieve this, but I think Bisq has some very talented individuals, so I am bullish this can be achieved. 

Aside from the development aspects mentioned is there anything you think that would help fuel growth?

**Comment by m52go** [comment]
@pazza83 thanks for the compliments. Bisq has many exceptional people contributing to it...my post is not meant to take anything away from any of them. I've had a wonderful time working with everyone.

But I think segmenting an API and daemon as mere development aspects is naive. 

Without ability to integrate, growth is directly hit from multiple points. As mentioned above, lack of integrations cut the project off from traction and exposure to new markets, new interfaces, and new use cases. Crucially, this also hinders community and contributor growth, since the only way to contribute to Bisq is to work on its big core code base (forced big+intimidating ramp-up time, forced Java, forced conventions...instead of hacking around on something new with full freedom). 

If we disregard the on-chain protocol for a moment, Bisq is incredible software that delivers a solid use-case for hundreds of people every day. The issue is that it's meant to be more of a protocol than a product. For a for-profit product stewarded by a traditional company, it would make sense to raise a big round of funding at this point to plow into marketing to drive growth to drive development. But for an open-source protocol, raising funding and doing traditional marketing isn't usually appropriate (or even possible), so integrations are needed. This is how protocols thrive. For Bisq these efforts have been on the horizon since before I ever started contributing years ago, but they still haven't worked out for one reason or another.

I haven't come up with any other ideas for order-of-magnitude growth. We have tried various educational, market-making, and publicity approaches over the past couple of years and while things have definitely improved, efficacy is limited. 

I don't doubt that Bisq can grow 10x in its current form, but it will take >10x the effort and >10x the time to do it this way.

**Comment by cbeams** [comment]
@m52go wrote:

> @cbeams please unassign me from this role.

I've unassigned you from this role and in turn removed you from the @bisq-network/team-leads, @bisq-network/growth and @bisq-network/growth-maintainers teams.

Echoing @pazza83, I'd like to say a big thanks for all you've done in this role! Your frustration is understandable, and while it's tough to see you resign, I appreciate that you've done it in a context of responsible criticism and integrity.

We need to consider how the duties of this role will be carried out (or not) going forward. The first one that comes to mind for me is approving Growth team members' compensation requests. For example, in this cycle (Cycle 30), you approved CRs for [me, @MwithM and @alexej996](https://github.com/bisq-network/compensation/issues?q=is%3Aissue+is%3Aopen+involves%3Am52go+). Has it been considered who will do this going forward?

> I have trimmed my other roles as well and generally expect to be less available across the project going forward.

To be clear, after this change, you remain assigned to the following roles:

```
$ gh issue list --repo bisq-network/roles --assignee m52go

Showing 5 of 5 issues in bisq-network/roles that match your search

#95  Community Manager            team:growth  about 4 days ago
#79  Keybase Admin                team:admin   about 1 year ago
#56  YouTube Admin                team:growth  about 1 day ago
#21  Twitter Admin                team:growth  about 1 day ago
#12  Website Operator/Maintainer  team:growth  about 1 day ago
```

Is that accurate from your perspective, @m52go?

**Comment by leo816** [comment]
@m52go Already said it privately, but just wanted to thank you for all that you've contributed to bisq as growth team lead.

**Comment by m52go** [comment]
@cbeams thanks for raising those points. You can unassign me from #95, as I just noted in the issue. #79 was taken over by @kbteamowner so I can be unassigned there as well. I'll hold on to the remaining 3 for now.

I had forgotten about compensation request reviews. There aren't too many of them. I'll discuss and find a replacement within the next week.

**Comment by ripcurlx** [comment]
@m52go Echoing again what's being said, thanks here officially for your great work for Bisq! I'm sorry about you getting frustrated because of the slow progress and I hope we can welcome you again, when it will be easier again to tap into potential growth opportunities. I do agree that we could be way ahead with the API already, if we would have found the right person for implementation earlier and had not to waste resources in a failed attempt. I think the API is making good progress (I use it myself already for trading purposes for a couple of months) and I think we should have a sufficient API coverage to be able to use Bisq without touching the UI in the not too distant future. Being a fully decentralized project is a challenge and it is a slow development approach, but it will pay off more and more in the future IMO. All the best for your future challenges Steve!

**Comment by flix1** [comment]
As proposed here https://github.com/bisq-network/growth/issues/287 and also discussed on matrix, I propose that this role be filled again in 2024. Growth Team Lead should have mandate and budget to build a team over the course of 2024.

They should develop a strategy, submit it for approval and do monthly reports as previously.

2 ideas:

1. Kickstart these 5 strategic countries
![image](https://github.com/bisq-network/roles/assets/4233784/890a99a5-4c32-4637-b9dc-67320a2ecfc1)

3bn+ people in those markets.

2. Increase XMR, privacycoin volumes by engaging with those communities and taking advantage of the probable delisting of privacycoins from many centralised exchanges in 2024. 
https://blockworks.co/news/crypto-exchanges-delisting-privacy-coins

Global XMR volume is 130M daily. 2M/day was a good day for Bisq in 2023. The growth potential is huge.

**Comment by Ven32123** [comment]
I am willing to help out.  I could possibly make user guides in PDF and video with subtitles to appropriate languages.

I still think making total transaction/mining/trade fee as low as possible is the way to go.
Second, we need to make the application bug free and streamed line for easy user experince.  I don't have any experince with coding so I can't help in this area.

**Comment by pazza83** [comment]
I am also interested in helping out with growth.

Outside of the dev heavy tasks like designing new protocols to reduce mining fees I think growth efforts should be focused on.

- Increase trades in identified target markets (preferably ones where inflation is a major issue). I think Bisq should look to recruit country specific contributors for this and give them a split of the trade fee revenue for these markets (giving a contributor say 50% of the trade fees that come from India for the first 2 years is better than Bisq keeping 100% of nothing). Also that way contribution payment is linked to results and easy to scale. 
- Add new fiat payment methods for more markets (Russian, Hong Kong, Nigeria etc).
- Re-opening Bisq to altcoins that wish to list and pay a fee (maybe changing the application process a bit). 
- With the increase in fees Bisq 1 should focus on bing a platform for larger value trades (I think this involves encouraging use for altcoin trading, XMR being the obvious example).  
- Marketing could be achieved from finding a contributor happy to go onto Bitcoin focused Podcasts to discuss Bisq.

I think having a clearly defined strategy for growth and associated budget would be a good starting point.

I would also like to link any spend on growth to a measurable increase in trade volume for the targeted currencies etc.

**Comment by solomon1923** [comment]
I would like to throw my hat in the ring for consideration of growth lead.

### Setting expectations:

I think it is important to address the fact that growth can come with some unwanted consequences. Bisq growths over time has been very organic this has allowed the platform, protocols, and the DAO to mature over time, thus improving Bisq’s decentralization. Bisq’s growth should continue to be organic to give users the freedom to buy and sell bitcoin with as little friction as possible. For this reason, I think Bisq’s growth initiatives should be as decentralized as possible. Comparing Bisq growth results to a centralized exchange is comparing apples to oranges.  The growth of Bisq will be slow over time. Bisq provides its users with an important tool, but currently it is one many people buying and selling bitcoin on centralized exchanges are unaware they need. Growth will come as more and more users start to experience friction with centralized services but the timescales for this are unknown and outside of Bisq’s control. Bisq should continue to build great products that enable as many people as possible to access the life rafts when needed. 

### Censorship resistance 

Bisq will achieve long term success if it is permissionless and resistant to censorship. As Bisq trade volume on Bisq increases it would also be good for Bisq to continue to work towards becoming as decentralized, permissionless and resistant to censorship as possible. This will take time to achieve, therefore, it makes sense for Bisq to expand horizontally into new markets. Especially those markets where inflation, currency controls, excessive regulations etc, make a service like Bisq even more needed. These are the true testing grounds for the need of Bisq. If Bisq is to be successful it should provide utility to users in places like; Venezuela, Argentina, China, India, Turkey and Russia.

### Promoting Bisq Easy to bitcoin buyers and sellers:

Bisq 2 will launch Bisq Easy early in 2024. If miner fees remain high in 2024 Bisq Easy will be the most cost-effective way for bitcoin buyers to acquire bitcoin on Bisq. For this to be a success it is important to establish both sides of the market place. Bisq has an existing user-base that will no doubt look to try out Bisq 2, but it will also appeal to a different demographic of users (eg no-coiners and sellers of lower value amounts of bitcoin), therefore, it is important that Bisq makes potential new users aware of Bisq 2.

### Developing and growing markets:

Bisq is primarily an exchange for people to buy or sell bitcoin for fiat. The majority of trades are fiat trades. At its heart Bisq seeks to replicate a ‘Satoshi Square’ a communal place people can come together to buy and sell bitcoin.   Bisq continues to develop products towards being a virtual meeting place for bitcoiners and no-coiners as this is at the core of its vision.  Bisq should continue to work towards this vision and develop volume in new fiat markets.  I think this will be best achieved by adding appropriate new payment methods and reviewing the bounty system for contributors in given locations Bisq would like to grow.

Despite the majority of trades being fiat trades, the majority of Bisq’s volume, and, therefore, revenue comes from XMR trading (approximately 70% of the bitcoin volume traded on Bisq in 2023 was for XMR). Bisq should look to deploy resources to develop and grow the XMR/BTC market.

Trading bitcoin for altcoins is antithetical to the values are many bitcoiners and, therefore, historically developing or adding new markets for altcoins on Bisq has been held back. New asset listing has been on hold for 4 years. The model for listing new assets was causing a lot of work for almost no reward. I propose to start a new discussion about creating a profitable model for listing new assets with a view to finding consensus from users and contributors.

### Digital first marketing:

Thinking again about Bisq being a ‘Satoshi Square’ it should focus its marketing efforts in digital communities where bitcoiners hangout. Whilst having a presence at in-person bitcoin person events has historical been part of Bisq’s marketing strategy I think in-person events are resource intensive and fail to scale. 

I think it is more important Bisq focuses on generating discussion on digital communication channels such as; Nostr, YouTube, Podcasts, Reddit, Twitter, etc. These are roles that could be completed by a decentralized team of contributors. 
The Bisq Blog is also currently neglected (over 3 years since the last post), I think efforts should be made to find a contributor to maintain it or else It should be retired. 

Bisq has historically avoided advertising, I think this should continue as it is not in line with an organic grassroots approach to growth.   

### Promoting BSQ:

Bisq tends to under-promote BSQ. However, I feel the current approach to, under-promote BSQ, is the correct one. Bisq should let BSQ find its own market price. It is fine for Bisq to encourage users to use BSQ if they would like to save of trade fees, and beyond that I do not think anything else other than improving usability and user understanding is required.

### Growth team budget:

I anticipate the growth budget being a maximum of $48,000 USD for 2024. This equates to approximately 10% profits or 4% of revenue for the calendar year 2023. This should cover all growth initiatives by the contributors that make up the growth team. The likelihood is it will be less than this as it will be going from a standing start so maybe half this amount ($24,000 USD) is more realistic in terms of costs for 2024.

Ideally, I would like contributions to be done by a growth team. Maybe something similar to the model of support with a few key contributors that can manage the ongoing requirements for social media accounts, blogs etc. 

### Success Metrics:

Success should ideally be linked to tangible results. 

An increase in trade volume can be measured in both BTC and USD.

Achieving growth in BTC volume terms would be fantastic but might be hard to achieve for a predominately fiat focused platform if bitcoin continues its march up in USD terms over time.

Achieving growth in USD volume would be more realistic. In 2023 Bisq users traded $110 million worth of bitcoin on the platform. It would be good to see this increase. A difficulty accessing this might be accessing trading statistics from Bisq 2 (It does not collect trade statistics from users).

Other ways to measure Bisq growth include:

- Total revenue 
- Total profit
- BSQ market cap increases
- Number of times the latest version of Bisq is downloaded  
- Increase in trade numbers and volume for a targeted currency  
- Increase in trade numbers and volume for a specific payment method

However, these factors have many more variables and making a direct correlation between contributor’s growth efforts and changes to the above becomes unclear.

**Comment by flix1** [comment]
Questions for @solomon1923 

Team: If you are selected to be growth team lead how will you recruit and organise team members? Would you have weekly team meetings? Would you have a monthly progress report with key metrics?

Budget: How would the budget be distributed between salaries (team lead compensation, team members compensation) and other projects, bounties, etc?

Committment: Is this going to be a full time job for you? Or part time? How many hours a week do you expect to dedicate to this role?

New markets: What is your strategy to kickstart Argentina, India, China, Nigeria? How would you find **market makers**? would you do **market days**? bounties?

XMR: how will you engage with the XMR community to find out what they want and perhaps attract contributors?

**Comment by solomon1923** [comment]
Thank you for the questions @flix1

> Team: If you are selected to be growth team lead how will you recruit and organise team members? Would you have weekly team meetings? Would you have a monthly progress report with key metrics?

In regards to recruitment I would follow a model similar to how it is done for the support team. Interested potential contributors could direct message me on Matrix or in the Growth room expressing their interest. Maybe there could be a similar role to [Support Intern](https://bisq.wiki/Support_Agent#Internship) for a Growth Intern. The user could have a cycle of trying out to be a growth intern. If successful the user could be made a Growth Agent. The process for recruitment would be as follows:

- Interested contributor messages me via direct message in Matrix, or posts their interest in the Matrix Growth room.  
- Initial text based interview (their ideas, previous experience, skills etc )
- Cycle as growth intern
- Proposal made as growth agent in the Proposal GitHub repo
- Weekly growth meetings
- Growth Lead to post monthly Growth report with key metrics (these are yet to be determined) 
- Growth Agents to post monthly individual reports for any compensation they are requesting
- Growth Lead to give feedback / recommend (or not) growth agents contribution requests

With regards recruitment for the roles I think users should be self selecting, as in they put themselves forward. I am aware that there has been some interest from multiple users in contributing for the growth team in the Matrix chat / GitHub discussions. That would be a good place to start.  

> Budget: How would the budget be distributed between salaries (team lead compensation, team members compensation) and other projects, bounties, etc?

My initial thoughts are about 10% of the budget would be for the team lead co-ordination role. This would be about $400 per cycle. The rest would be for work done for example:

- Bounties claimed for growth in new markets
- YouTube videos created and posted on Bisq channel
- Blog posts
- Podcasts appearance
- Reddit marketing
- Nostr marketing
- Twitter marketing
- Bisq 2 marketing
- New payment method creation 
- Specific currency marketing work
- Monero marketing work

> Committment: Is this going to be a full time job for you? Or part time? How many hours a week do you expect to dedicate to this role?

I think the co-ordination role would be part time. I would expect to put in a minimum of 5 hours a week into the role. The bulk of the cost for the work will contributions delivered. I would anticipate that this would be done collectively by the Growth Team. Whilst there might be some aspects of the role I take on myself, I would like to have multiple contributors delivering the required work. This will enable contributions to match a users strengths, interests, and local knowledge. 

> New markets: What is your strategy to kickstart Argentina, India, China, Nigeria? How would you find **market makers**? would you do **market days**? bounties?

All the results-to-date with these markets have been disappointing. My initial goal will be to find out what strategies work for new markets.

I would like to retire the existing bounties and create new ones for each of the above markets. My strategies for this would be:

- Have a local contributor for the local market that is able to market make
- Link payments to be USD equivalent in BSQ rather than a fixed amount in BSQ to account for volatility 
- Payments will be linked to market making activity (multiple buy and sell offers within 10% spread; eg buy 5% under and sell 5% over)
- Educate contributors how to make money market making, ideally you want it to be sustainable long term without any bounty needed
- Give the market makers a percentage of all trade fees for that market eg Growth contributor for India gets 0.8% of all trades that take place in that market in 12 months. Bounty is claimed after the 12 month period, and subject to the contributor keeping offers online.  
- Contributors could also post, create video, arrange market days, blog etc to promote Bisq in the language of their country in places other bitcoiners from their country are  found. These contributions would be compensated per cycle.
- Give the responsibility of growing local markets to a specific contributor, and share the success of the growth with them as a bounty.

> XMR: how will you engage with the XMR community to find out what they want and perhaps attract contributors?

Some of the research has already been completed: https://github.com/bisq-network/proposals/issues/349#issuecomment-1004297559

Some initial thoughts about XMR growth are:

- One of the frustrations of XMR users is; lack of ability get get XMR in their country, freezing of funds, shotgun KYC, by CEX and swap services. Bisq's 2 of 2 multi-sig protocol is permission-less. This should be promoted in relevant places; Monero subreddit, Monero Matrix rooms, Relevant Dread forums, Monero.town etc
- Bisq has current protocols that make trading over 0.5 BTC worth of XMR risky. This should be addressed to make large traders able to use Bisq with less risk of ending up being reimbursed in BSQ.
- Some XMR maxi's would not like to trade using BTC as the multisig. Maybe a future possibility is a Bisq Easy style trade protocol for XMR <-> fiat. This would be subject to DAO consensus. 
- Likely there will be competition for XMR/BTC atomic swaps. Bisq should be in this competition and atomic swaps should be promoted in the relevant places.
- Make relevant Community Crowdfunding System (CCS) proposals https://ccs.getmonero.org/
- Publish relevant Monero Bounties: https://bounties.monero.social/
- Bisq should be seen as an essential tool for the Monero community just as it is for the Bitcoin community 

Another important factor for promoting Bisq in the Monero community is the possibility to attract some developers familiar with the monero code base. Developers with these skills would be much needed for Bisq if it wants to expand on the protocols that can be used by monero users. 

Growing the XMR/BTC user base of Bisq is the biggest opportunity to make the growth team really pay for itself. However it is also creates the possibility of attracting unwanted attention. These opposing consequences should be taken into consideration.

**Comment by Ven32123** [comment]
@flix1 @solomon1923 

Team: I would be willing to assist any growth team lead in areas where I have expertise.  In my opinion, weekly meetings could be difficult  depending on team member's time zone and other scheduled commitments.  I think meetings should be scheduled prior to key projects to strategize and distribute work.  Also, meetings prior to important deadlines may be better than having it weekly.  I think monthly progress reports would be great to show results and areas where improvement can be made.

Budget: I would request compensation based on value provided compared to allowed budget.

Commitment: I would be looking to do this part-time, so I would be willing to spend 1-10 hours per week depending on what needs to be done.

New markets:  The best way is to find and educate people to become market makers for these areas. Revenue generated in fees by these market makers  can have a % sent back as a reward.  We would need to find these market makers on Reddit, forums, twitter, YouTube or what ever social media that is most used in that emerging market.

XMR/alt coins: I would like to add other coins and trading pairs and expand bisq similar to Uniswap or other decentralized exchange.   Goal should be to make transaction fees as low as possible.  Only add coins if there are people willing to be a market maker for that trading pair.  Reward incentive can be a % of revenue generate being sent back to the market maker.

**Comment by Jayboi23** [comment]
@flix1 
In my current role as Bisq Nostr account manager, I have consistently demonstrated a keen understanding of market dynamics, particularly in identifying issues related to liquidity. Recognizing that low liquidity can hinder growth prospects, I have developed a comprehensive approach to address this challenge head-on.

* **Content marketing**: We can develop high-quality content that educates readers about the benefits of using Bisq and decentralized exchanges in general (which have been doing on Nostr since my recruitment till date). We can also create blog posts, articles, and tutorials that explain how Bisq works, its security features, and the advantages it offers over centralized exchanges. Share this content on relevant platforms such as cryptocurrency blogs, forums, social media, and industry publications to attract readers.
* **Social media presence**: As growth team, we can establish a strong presence on social media platforms that are popular among cryptocurrency enthusiasts. Share informative and engaging content about Bisq, including updates, new features, and success stories. Actively participate in relevant discussions, answer questions, and engage with the community to build trust and drive curiosity about Bisq.
* **Influencer partnerships:** I’ll suggest we collaborate with influential individuals in the cryptocurrency space, such as prominent bloggers, vloggers, or industry experts. They can review Bisq, mention its advantages, and recommend it to their followers. These partnerships can significantly increase awareness and drive traffic to Bisq.
* **Community engagement**: Engage with the cryptocurrency community by participating in forums, chat groups, and online communities where traders and enthusiasts gather. By sharing insights, answering questions, and providing support. By being an active and helpful community member, we can position Bisq as a reliable and valuable platform, encouraging readers to check it out.
* **Paid advertising**: We can also consider using targeted online advertising to reach a broader audience. Platforms like Google Ads and social media advertising platforms allows their users to create highly targeted campaigns based on demographics, interests, and keywords. Craft compelling ad copy and visuals that highlight Bisq's unique features and benefits to entice readers to visit the platform.
* **Referral programs**: Implement a referral program that incentivizes existing Bisq users to invite their friends and contacts to join the platform (could be inform of bounties). We can offer rewards or discounts for successful referrals, creating a win-win situation where existing users are motivated to spread the word about Bisq, while new users are enticed to give it a try.
* **Community events and meetups**: Organize or participate in cryptocurrency-related events, conferences, or meetups. These gatherings provide opportunities to showcase Bisq, deliver presentations, and network with potential users and industry professionals. By demonstrating the value and functionality of Bisq in person, you can create lasting impressions and attract interested readers to explore the platform.

The listed points above are the areas I think we should focus more on while other plans can follow in the long run. I will open to work with the growth members the best way I can in other to make Bisq well recognized and increase Liquidity in the low liquidity markets

**Comment by aodalaeon** [comment]
Dear Bisq team,
I'm late to the discussion, but want to express my motivation to contribute to Bisq Growth. Salokiam informed me about this opening, I was at the Prague meeting with him last summer. My fiat job is B2B sales, so I have experience in marketinig, cold calls etc. 
I joined the Bisq Growth channel in Matrix, kindly find me there (aodalaeon).

**Comment by solomon1923** [comment]
Proposal for myself to assume the growth lead role was accepted. I will request access to the growth repo

**Comment by cbeams** [comment]
Cheers, @solomon1923! Per the approval of https://github.com/bisq-network/proposals/issues/448 and your request at https://github.com/bisq-network/roles/issues/28#issuecomment-1957524715, I've just assigned you as the owner of this role. Please go ahead with cycle reports here on this issue, and all the best to you in this role!

**Comment by solomon1923** [comment]
## Cycle 57 Report

Recently started in the growth lead role. It has started slowly as expected. 

Bisq being decentralized means it is difficult to get a handle on who is doing what. For example who is taking ownership of Bisq's accounts on; Matrix, Twitter, Nostr, Facebook, YouTube, website, blog etc. Some of the account users appear to abandoned the role which makes contacting the user or gaining access difficult. I am also un-sure how much access to accounts should be centralized, but not having this information is a barrier to coordinating a growth strategy. 

My current thinking is to create a bonded role for a 'Community Admin' that would at least know who is responsible for the account and how to access or transfer the accounts if needed. This will likely take some time to establish.

In the meantime I have been working with the growth team, which has consisted of users that have expressed an interest in contributing to assisting with Bisq's growth efforts. It is early days and I anticipate this changing over time. Ideally a growth team would be more effective if there was an agreed growth strategy. 

Some of the growth work to date I have done includes: 

- Held a successful AMA on stacker.news [Bisq 2 has just been released. I am a Bisq contributor AMA about Bisq or Bisq 2](https://stacker.news/items/469058)
- Completed [three calls with growth team](https://github.com/bisq-network/growth/issues/289)
- Created [Bounty for users wanting to appear on podcasts to represent Bisq](https://github.com/bisq-network/growth/issues/290)
- Started discussion to [define a Nostr strategy for Bisq](https://github.com/bisq-network/growth/issues/291)
- Started to [resolve issues with Encrypted chat in Matrix General Room](https://github.com/bisq-network/bisq/issues/7026)

I expected this role to be challenging and it is. I think it will take me a few cycles to find my feet 
so to speak before pushing on with growth. if anyone would like to help please reach out to me on Matrix, you can find me hanging about in the growth room.

**Comment by solomon1923** [comment]
## Cycle 58 Report

Not much to report this cycle in terms of work delivered. 

With the events of the recent weeks it seems evident that how you can communicate to people on your website  / socials can be used against you by the powers that be. Therefore, I think it is prudent to consider some things Bisq should avoid.

I will have a think of what Bisq should, and should not, be communicating over the coming cycle.

**Comment by solomon1923** [comment]
## Cycle 59 Report

Not much to report this cycle in terms of work delivered.

Spoke with some contributors about what direction Bisq growth and marketing should take.

I have asked for the Nostr marketing to be put on hold, with this cycle being the last one to be compensated. Thanks to @Jayboi23 for all his work on growing this account.

Will update when I have more information.

**Comment by solomon1923** [comment]
## Cycle 60 Report

Following disucssion with team members it has been decided that the Bisq Growth team will be retired. There is, therefore, no longer a requirement for a Growth Lead.

I am resigning from the role of Growth Lead with immediate effect.

I am will still continue to be involed with Bisq. I will keep my current role as a Bisq 2 mediator and also look to contribute in any other areas I feel I can be of use.

**Comment by BlackhatEspeed** [comment]
I will pickup where solomon1923 left off. I'm a google analytics expert. Are there tutorial videos on management operations? I will create a managerial set of variables for MSB Banking in Bisq as I believe it's possible. I will make them into training videos so that larger amounts could be hosted in BTC as the market has grown and I'm at 300 BTC.

**Comment by pazza83** [comment]
@BlackhatEspeed Thanks for the offer. Unfortunately this role has now been retired.
